### PR TITLE
Removes Bay

### PR DESCRIPTION
**Pull request by MarinaGryphon**
* Resets the changelog.
* Removes some now-unused files.
* Ports some Travis tests from Aurora, ESPECIALLY ONE TO CHECK FOR MAPMERGING, @ProbablyCarl THIS ONE IS FOR YOU.
* Replaces, like, every instance of Baystation12 with BastionofHestia.
Will probably break things so don't merge yet.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -50,4 +50,4 @@
 - [ ] Issue could be reproduced by different players
 - [ ] Issue could be reproduced in multiple rounds
 - [ ] Issue happened in a recent (less than 7 days ago) round
-- [ ] [Couldn't find an existing issue about this](https://github.com/Baystation12/Baystation12/issues)
+- [ ] [Couldn't find an existing issue about this](https://github.com/BoHBranch/BoH-Bay/issues?state=open)
```
Comment (Roland410) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
- [ ] [Couldn't find an existing issue about this](https://github.com/BoHBranch/BoH-Bay/issues)
```

File: config/example/motd.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
-<h1>Welcome to Space Station 13!</h1>
+<h1>Welcome to Bastion of Hestia!</h1>
 
--<i>This server is running <a href="http://baystation12.net/">Baystation 12's</a> modification of the <a href="http://ss13.eu/">/tg/station13</a> SS13 code.</i>
+-<i>This server is running <a href="http://github.com/BoHBranch/BoH-Bay/">Bastion of Hestia's</a> modification of the <a href="http://baystation12.net/">Baystation12</a> SS13 code.</i>
 <br><br>
-<strong>Bugtracker:</strong> <a href="http://baystation12.net/forums/threads/issue-tracker-report-template.110/">for posting of bugs and issues.</a>
+<strong>Bugtracker:</strong> <a href="https://github.com/BoHBranch/BoH-Bay/issues?labels=Bug&state=open">for posting of bugs and issues.</a>
```
Comment (Roland410) [inline config/example/motd.txt]:
```suggestion
<strong>Bugtracker:</strong> <a href="https://github.com/BoHBranch/BoH-Bay/issues">for posting of bugs and issues.</a>
```

File: html/templates/header.html [hunk 3]
```diff
@@ -23,27 +23,26 @@
 <table align='center' class="top">
 	<tr>
 		<td valign='top'>
-			<div align='center'><font size='3'><b>Baystation 12</b></font></div>
+			<div align='center'><font size='3'><b>Bastion of Hestia</b></font></div>
 			<div align='center'><font size='3'><b><i>A Space Station 13 Project</i></b></font></div>
 			
-			<p><div align='center'><font size='3'><a href="http://wiki.baystation12.net/">Wiki</a> | <a href="https://github.com/Baystation12/Baystation12/">Source code</a></font></div></p>
+			<p><div align='center'><font size='3'><a href="https://github.com/BoHBranch/BoH-Bay">Source code</a></font></div></p>
```
Comment (Roland410) [inline html/templates/header.html]:
```suggestion
			<p><div align='center'><font size='3'><a href="http://wiki.baystation12.net/">Wiki</a> | <a href="https://github.com/BoHBranch/BoH-Bay">Source code</a></font></div></p> //Change this once a proper wiki is up.
```
Until we have a wiki that has the information and shit, you don't want to get rid of such in my opinion.

File: html/templates/header.html [hunk 4]
```diff
@@ -23,27 +23,26 @@
...
-			<font size='2'><b>Special thanks to:</b> The developers of /tg/station, /vg/station, GoonStation, the original Space Station 13, and BYOND</font>
-			<div align='center'><font size='2' color='red'><b><br>Have a bug to report?</b> Visit our <a href="https://github.com/Baystation12/Baystation12/issues?labels=Bug&state=open">Issue Tracker</a>.<br></font></div>
+            <font size='2'><b>Developers:</b> Kerbal22, MoondancerPony<br></font>
+			<font size='2'><b>Currently Active GitHub contributor list:</b> <a href='https://github.com/BoHBranch/BoH-Bay/graphs/contributors'>-Click Here-</a><br></font>
+			<p><font size='2'><b>Credits:</b> BurgerBB, NothingButCarl, HaltManZ<br></p>
+			<font size='2'><b>Special thanks to:</b> The developers of Baystation12, /tg/station, /vg/station, GoonStation, the original Space Station 13, and BYOND</font>
+			<div align='center'><font size='2' color='red'><b><br>Have a bug to report?</b> Visit our <a href="https://github.com/BoHBranch/BoH-Bay/issues?labels=Bug&state=open">Issue Tracker</a>.<br></font></div>
```
Comment (Roland410) [inline html/templates/header.html]:
```suggestion
			<div align='center'><font size='2' color='red'><b><br>Have a bug to report?</b> Visit our <a href="https://github.com/BoHBranch/BoH-Bay/issues">Issue Tracker</a>.<br></font></div>
```

File: tools/mapmerge2/README.md [hunk 5]
```diff
@@ -15,28 +15,15 @@ contains the desired changes.
...
+To install Python dependencies, run `requirements-install.bat`, or run
+`python -m pip install -r requirements.txt` directly. See the [Git hooks]
+documentation to install the Git pre-commit hook which runs the map merger
+automatically, or use `tools/mapmerge/Prepare Maps.bat` to save backups before
+running `mapmerge.bat`.
 
 For up-to-date installation and detailed troubleshooting instructions, visit
 the [Map Merger] wiki article.
 
-## Usage
```
Comment (Roland410) [inline tools/mapmerge2/README.md]: This probably should be kept for people who use the mapmerger manually like I do.

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (MarinaGryphon) [reply .github/ISSUE_TEMPLATE.md]: If they're reporting an issue and a similar issue is closed, it's either:
* Not relevant due to new information
* Falsely closed due to an incomplete/bad bugfix, or
* Only superficially similar.
wait nvm doesn't it default to open anyway i'm dumb

File: html/templates/header.html (hunk 3 above)
Comment (MarinaGryphon) [reply html/templates/header.html]: I was told the Discord would suffice, but I kind of agree.

File: tools/mapmerge2/README.md (hunk 5 above)
Comment (MarinaGryphon) [reply tools/mapmerge2/README.md]: smh you're bad, but uh. okay! i'll keep that in then

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (Roland410) [reply .github/ISSUE_TEMPLATE.md]: Yes, yes it does.

File: html/templates/header.html (hunk 3 above)
Comment (Roland410) [reply html/templates/header.html]: Discord means people would be flooding with HOW TO SURGERY and other problems there are answers already written for.

File: tools/mapmerge2/requirements.txt [hunk 6]
```diff
@@ -1,3 +1,2 @@
-pygit2>=0.27.2
-bidict>=0.13.1
-Pillow>=5.1.0
+pygit2==0.26.0
```
Comment (Roland410) [inline tools/mapmerge2/requirements.txt]: By the way, is it intended to now work with an older version apprently?

**Comment by BurgerLUA** [comment]
@MarinaGryphon Are you still working on this? The PR is kinda getting old.

**Comment by BurgerLUA** [comment]
Hasn't been worked on for a while and is very stale. Please reopen this PR when you have done work on it.

**Pull request closed without merging**